tests/OSPSuite.Core.IntegrationTests/DiffBuilders/ContainerDiffBuilderSpecs.cs

```diff
@@ -96,4 +98,23 @@ public void should_have_set_the_parent_to_the_common_3rd_level()
          _report.ElementAt(0).CommonAncestor.ShouldBeEqualTo(_c111);
       }
    }
+
+   public class When_comparing_two_container_having_different_parentpath : concern_for_ObjectComparer
+   {
+      protected override void Context()
+      {
+         base.Context();
+         _comparerSettings.OnlyComputingRelevant = false;
+
+         _object1 = new Container { Name = "O1", ParentPath = new ObjectPath("A", "B") };
+         _object2 = new Container { Name = "O1", ParentPath = new ObjectPath("A", "C") };
+      }
+
+      [Observation]
+      public void should_return_a_report_containing_the_container_path_difference()
+      {
+         _report.Count().ShouldBeEqualTo(1);
+         Assert.IsTrue(_report.First().Description.StartsWith("Parent Paths are not equal"));
```

Comment:
ok this is it. One last change. WE never use the ASsert but use our BDD stuff
can you do something along the line of _report.First().....ShouldBeEqualTo 
and remove the ref to NuNit at the top. 
also is there a constant that you can use instead of the magic string Parent Paths are not equal? Are we typically comparing to strings in the tests ?

Reply:
Sure thing, just did          _report.First().Description.StartsWith("Parent Paths are not equal").ShouldBeTrue();
which is in line with our asserts.

Great catch mate!